src/test/java/org/openmrs/addonindex/scheduled/FetchCoreVersionsListIT.java

```diff
@@ -0,0 +1,101 @@
+package org.openmrs.addonindex.scheduled;
+
+import io.searchbox.client.JestClient;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.openmrs.addonindex.domain.AddOnInfoAndVersions;
+import org.openmrs.addonindex.domain.AddOnType;
+import org.openmrs.addonindex.domain.AddOnVersion;
+import org.openmrs.addonindex.domain.artifactory.ArtifactoryFolderInfoChild;
+import org.openmrs.addonindex.domain.artifactory.VersionList;
+import org.openmrs.addonindex.service.ElasticSearchIndex;
+import org.openmrs.addonindex.service.artifactory.VersionsService;
+import org.openmrs.addonindex.util.Version;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.boot.web.client.RestTemplateBuilder;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.IsInstanceOf.instanceOf;
+import static org.junit.Assert.assertThat;
+import static org.mockito.BDDMockito.given;
+
+@SpringBootTest
+@RunWith(SpringRunner.class)
+public class FetchCoreVersionsListIT {
```

Comment:
This class is not actually testing what it's supposed to be testing.

What you want to do here is to call the scheduled task, and then make sure it did the right thing. (In this case that means verifying that it called VersionService.setVersions(...), or whatever that method is called.)